triplea_maps.yaml

```diff
@@ -374,6 +374,8 @@
     <br>Includes: Original/OOB and Second Edition and a Global 1942 mod.
     <br>Now also includes Balanced Mod and variants of Balanced Mod and 2nd ed with Combat Move first. Ozteas 1941 setup is now included.
     <br>
+    <br>Fixed oztea 41 so all original Duth Islands will be owned by whoever controlls them.
+    <br>
```

Comment:
@DanVanAtta @RoiEXLab @ron-murhammer Do we normally include changelog-type information like bug fixes in map descriptions?  I don't think I've ever seen that before.

Reply:
I don't think so, but I'm normally not doing any map related stuff so...

Reply:
Not in the yaml file. These would belong in the actual map change log in the game XML.

Reply:
so should I just bump the version and get rid of the description ? Can't change the change log number. I guess I could add to the change log with the date ? Would have to do another PR as ssoloff already merged it.

Or should I just not worry about the description ? Or put in the comment space on this PR ?

Reply:
> so should I just bump the version and get rid of the description ?

Yeah, it sounds like this PR should just be a one-line change to the `version` property.

> Can't change the change log number. I guess I could add to the change log with the date ? Would have to do another PR as ssoloff already merged it.

I looked through the history of the other WW2 Global XML files, and none of the changes were accompanied by a change log update.  @panther2 Can you confirm that, AFAYK, no one has been updating the change log since 3.9 was released?

Reply:
@ssoloff I cannot confirm that it hasn't been updated since the release of 3.9. But it looks like updating the 3.9 notes stopped at a later time (years ago). The BM xml files include some BM related stuff in the notes. But the "core" game notes are the same as in the other Global files, too.
Even the change from Global-Alpha3.9 to Global-2nd Edition has not been reflected in the Game notes. Maybe that caused that no one felt responisble for updating neither the map version nor the game notes.
We should really implement a sound versioning system.